camera_matrix:
  fx: 1738.93160284725
  fy: 1739.91485948122
  cx: 799.591608135606
  cy: 602.953914087021
dist_coeffs: [0.013335801403273,-0.172291569685205, 4.9476489912055e-05, -4.01260069663971e-05, 0.617680184137795]
